pytorch_pfn_extras/dataset/tabular/_transform.py

```diff
@@ -3,18 +3,19 @@
 
 
 class _Transform(tabular_dataset.TabularDataset):
-    def __init__(self, dataset, keys, transform, fetch_keys=None):
-        if not isinstance(keys, tuple):
-            keys = (keys,)
-
+    def __init__(self, dataset, keys, transforms):
         self._dataset = dataset
+
+        key_set = set()
+        for s, _ in transforms:
+            if any([k in key_set for k in s[1]]):
+                raise ValueError('Transformations must be disjoint')
+            key_set.update(s[1])
+        if len(key_set - set(keys)) != 0:
+            raise ValueError('Transformations must produce all specified keys')
```

Comment:
The condition and message are mismatched.
The condition indicates that all produced keys by transformations are specified in keys.
If `keys = ('A', 'B')` and transformations produce `('A',)`, this check passes.

Reply:
Changed it!